cmis_web_proxy/controllers/cmis.py

```diff
@@ -130,6 +131,20 @@ def _cmis_proxy_base_url(self):
         return urllib.parse.urljoin(request.httprequest.host_url,
                                     CMIS_PROXY_PATH)
 
+    def _get_requests_timeout(self):
+        """
+        Fetch and return the requests timeout (ir.config_parameter in seconds)
+        Trigger a logged error if something is wrong with the ir.config_parameter
+        :return: timeout (seconds in float)
+        """
+        try:
+            timeout = safe_eval(self.env['ir.config_parameter'].get_param(
+                "cmis.requests_timeout", "FAULTY"))
```

Comment:
Let's avoid save_eval when we can. Better usr `int()` ... `except ValueError`

Reply:
or `float()`

Reply:
The reason was that the timeout can also be a tuple of floats (connect timeout and read timeout)

Reply:
Then the docstring need updating ;) But is that not too fancy ? Is this tuple of timeout something that the requests library supports ? And if we want to use the same config value in the cmis module that uses cmislib, will it support that too ?

Reply:
Maybe it's too fancy but it comes from the requests library when i checked the type of the timeout ;-)
Don't know for cmislib (and did not find any timeout support at all) so we'll keep it simple and i'll follow your wisdom.